drivers/tee/optee/optee_smc.h

```diff
@@ -383,19 +383,19 @@ struct optee_smc_disable_shm_cache_result {
 	OPTEE_SMC_RPC_VAL(OPTEE_SMC_RPC_FUNC_FREE)
 
 /*
- * Deliver an IRQ in normal world.
+ * Deliver a foreign interrupt in normal world.
```

Comment:
> Doesn't "Deliver a foreign interrupt in normal world" sound better?

Hi @jenswi-linaro ,
Sorry, did you suggest to reword the comment "Deliver a foreign interrupt in normal world."